src/encoded/schemas/individual_human.json

```diff
@@ -57,6 +57,13 @@
                 "postnatal"
             ]
         },
+        "protected_data": {
+            "title": "Protected Data",
+            "description": "A flag to indicate whether consent to publish sequence files from this individual is not available. If true, fastq and bam files are subject to restricted access.",
+            "type": "boolean",
+            "default": false,
```

Comment:
I wonder if it might make sense to have the default be true?  Safer that way and going forward it will require deliberate change for any wrangler creating a human individual?